external/llvm-project/mlir/include/mlir/Dialect/GPU/GPUOps.td

```diff
@@ -1120,8 +1121,9 @@ def GPU_MFMAOp:
   let description = [{
     The `gpu.mfma` op is an abstraction of XDLOPS.
   }];
-  let parser = [{ return parseMFMAOp(parser, result); }];
-  let verifier = [{ return ::verify(*this); }];
+  let assemblyFormat = [{
+    `(` $sourceA `,` $sourceB `,` $destC `)` attr-dict `:` type($sourceA) `,` type($destC)
```

Comment:
should also encode type($sourceB)

Reply:
`type($sourceA)` is always the same as `type($sourceB)`, so I think this can be omitted.